src/ChipMachine_commands.cpp

```diff
@@ -256,6 +256,26 @@ void ChipMachine::setupCommands()
         player.nextSong();
     });
 
+    cmd("favorite_shuffle", [=]() {
+        toast("Favorites shuffle!");
+
+        auto playlistPath =
+            (Environment::getConfigDir() / "playlists" / currentPlaylistName).string();
+        std::vector<SongInfo> target = 
+            MusicDatabase::getInstance().getPlaylist(playlistPath);
+
+        // TODO: Switch to std::shuffle?
+        std::random_shuffle(target.begin(), target.end());
+
+        player.clearSongs();
```

Comment:
Hmm. FWIW, this is copypasta code from `shuffleSongs` in ChipMachines_keys.cpp. Might be better to add a `bool favorites` or playlist name param to that function and move the playlist shuffle over there